cocaine/proxy/proxy.py

```diff
@@ -290,6 +291,12 @@ def __init__(self, locators=("localhost:10053",),
         # it's used to reply on `ping` method
         self.locator_status = False
 
+        self.unicorn = Service("unicorn", locator=self.locator)
```

Comment:
The name can be changed, so it can be useful (not blocking ATM) to move this hardcoded value into config.